main.py

```diff
@@ -0,0 +1,41 @@
+import random
+from art import logo
+
+# level_of_difficulty() returns number of attempts based on difficulty selection
+def level_of_difficulty(choice):
+    if choice == 'easy':
```

Comment:
In general, I strongly prefer to see dictionaries / mappings instead of if statements.  Please use a dictionary here instead.